org.eclipse.ice.bats/README.md

```diff
@@ -0,0 +1,122 @@
+# BATS
+
+BATS, the Basic Artifact Tracking System (BATS), is a simple data management service for managing scientific data.
+
+BATS is a standalone maven package that can be used within or outside of ICE.
+
+BATS leverages [Apache Jena](https://jena.apache.org/index.html) to create [RDF Models](https://jena.apache.org/tutorials/rdf_api.html) and connects to [Apache Jena Fuseki](https://jena.apache.org/documentation/fuseki2/index.html) to publish these models.
+
+
+
+## Build Instructions
+
+### Prerequisites
+
+BATS requires a full installation of Docker for building, executing, and storing images.
+
+
+### Using Maven
+
+Building the package can be performed like most other maven package, where the end result is a jar file that can be included as a dependency
```

Comment:
...like most other Maven packages... (currently missing the s)

Reply:
Fixed!